### ISSUE TITLE
Improve routing (help wanted)

### ISSUE THREAD

**Issue by nikgraf** [nikgraf/CarteJaune#7]
Currently using a [custom baked solution](https://github.com/nikgraf/CarteJaune/blob/master/reducers/activeRoute.js). I guess there is something better this …

**Comment by nikgraf** [comment]
Might be a good read: https://medium.com/the-exponent-log/routing-and-navigation-in-react-native-6b27bee39603#.66648kihn

**Comment by sibeliusseraphini** [comment]
Maybe this one: https://github.com/aksonov/react-native-router-flux

**Comment by nikgraf** [comment]
@sibeliusseraphini looks good to me!

**Comment by sibeliusseraphini** [comment]
Yeah, take a look on my Boilerplate https://github.com/sibeliusseraphini/react-native-redux
